bode/bode/models/tag.py

```diff
@@ -38,5 +31,11 @@ def delete(tag_id):
 
         return tag
 
+    def getByName(tag_name):
+        tag = Tag.query.filter(Tag.name == tag_name).first()
+        if tag is None:
+            raise NoResultFound
+        return tag
```

Comment:
[.one()](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.one) will throw the error for you
```suggestion
        return Tag.query.filter(Tag.name == tag_name).one()
```

Reply:
Yeah, actually this throw was unnecessary